AppCenter/AppCenter/MSACAppCenter.h

```diff
@@ -71,61 +72,36 @@
 + (void)startFromLibraryWithServices:(NSArray<Class> *)services;
 
 /**
- * Check whether the SDK has already been configured or not.
- *
- * @return YES if configured, NO otherwise.
+ * The flag indicates whether the SDK has already been configured or not.
  */
-+ (BOOL)isConfigured;
+@property(class, atomic, readonly, getter=isConfigured) BOOL configured;
```

Comment:
Do we need `NS_SWIFT_NAME` here?

Reply:
Hmm... We can add it in case if someone thinks of accessing it, yeah